pkg/tempopb/tempo.proto

```diff
@@ -190,6 +194,11 @@ message SpanMetricsRequest {
   uint64 limit = 3;
 }
 
+message SpanMetricsSummaryRequest {
```

Comment:
Do we want to also include a `limit` here?  Thinking that we could pass it directly to the queriers.  Alternatively, a limit could be imposed by a querier config setting, which is then used during the call to the generators.

Reply:
Yes let's expose it at least to let us experiment with scaling.  Agree it could eventually operate like `SearchRequest.Limit` with a server-side default and maximum.